contracts/handlers/ERCHandlerHelpers.sol

```diff
@@ -41,19 +45,30 @@ contract ERCHandlerHelpers is IERCHandler {
         _;
     }
 
+    modifier onlyAuthorized() {
+        _onlyAuthorized();
+        _;
+    }
+
     /**
         @param bridgeAddress Contract address of previously deployed Bridge.
      */
     constructor(
         address          bridgeAddress
     ) {
         _bridgeAddress = bridgeAddress;
+        _setupRole(DEFAULT_ADMIN_ROLE, msg.sender);
+        _setupRole(LIQUIDITY_MANAGER_ROLE, bridgeAddress);
     }
 
     function _onlyBridge() private view {
         require(msg.sender == _bridgeAddress, "sender must be bridge contract");
     }
 
+    function _onlyAuthorized() private view {
+        if(!hasRole(LIQUIDITY_MANAGER_ROLE, _msgSender())) revert NotAuthorized();
```

Comment:
Do we want the admin as well as bridge to have the liqudity manager role?
Also, there probably would be cases where we don't want the bridge to have that role.

Reply:
@MakMuftic Can you specifiy cases again in writing so we are all on the same page please.